cosmwasm/contracts/pyth/src/contract.rs

```diff
@@ -61,12 +64,21 @@ pub fn instantiate(
 ) -> StdResult<Response> {
     // Save general wormhole and pyth info
     let state = ConfigInfo {
-        owner:             info.sender,
-        wormhole_contract: deps.api.addr_validate(msg.wormhole_contract.as_ref())?,
-        data_sources:      HashSet::from([PythDataSource {
+        owner:                      info.sender,
+        wormhole_contract:          deps.api.addr_validate(msg.wormhole_contract.as_ref())?,
+        data_sources:               HashSet::from([PythDataSource {
             emitter:            msg.pyth_emitter,
             pyth_emitter_chain: msg.pyth_emitter_chain,
         }]),
+        chain_id:                   msg.chain_id,
```

Comment:
added governance fields to ConfigInfo and the initialization message. 

Note that `fee_denom` below seems to be needed to determine which coin the fees are paid in. I don't totally understand what this does, but I'm copying from wormhole here https://github.com/wormhole-foundation/wormhole/blob/main/cosmwasm/contracts/wormhole/src/contract.rs#L74

Reply:
We should think a little bit about it and we should ask WH people how they handled it. Basically in cosmos there might be more than 1 native transaction, as in terra there were both LUNA and UST. 

So the question is that do we want to have only one accepted coin per each chain?

Reply:
wormhole only supports one coin (`uluna`). We can support multiple, but let me add that in a separate pr. (I think we'll need more governance instructions if we go that route.)